### ISSUE TITLE
Update `focus` color to meet color contrast on backgrounds

### ISSUE THREAD

**Issue by taysea** [grommet/hpe-design-system#4434]
Our current focus indicator outline does not meet 3:1 contrast on our background colors. We will update it to a color that does.

Deliverable:
- [x] Updated focus color for light and dark mode

**Comment by taysea** [comment]
A new color has been added
